opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -239,7 +239,7 @@ class SimulatorFullyImplicitBlackoilEbos
 
             // write the inital state at the report stage
             if (timer.initialStep()) {
-                output_writer_.writeTimeStep( timer, state, well_state, solver->model() );
+                //output_writer_.writeTimeStep( timer, state, well_state, solver->model() );
```

Comment:
does it still write the initial solution? If yes, it's better remove the whole `if`?